staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -526,6 +529,25 @@ const (
 	PersistentVolumeClaimFileSystemResizePending PersistentVolumeClaimConditionType = "FileSystemResizePending"
 )
 
+type PersistentVolumeClaimResizeStatus string
+
+const (
+	// When expansion is complete, the empty string is set by resize controller or kubelet.
+	PersistentVolumeClaimNoExpansionInProgress PersistentVolumeClaimResizeStatus = ""
```

Comment:
Why empty string and not nil?

Reply:
So my original idea was, I thought I needed an invariant which I can use as default (without risking dereferencing a `nil`). This would also serve as a terminal condition when resizing is complete (for informing user). But I see how this can be problematic, I have marked this as a follow up item - https://github.com/kubernetes/kubernetes/issues/106429 before the feature goes beta.